### DISCUSSION TITLE
cjs and mjs css file

### DISCUSSION THREAD

**Discussion by FelixBlaisThon** [mantinedev/mantine#5547] [Q&A]
Hello, I was trying to understand how you were able to create files of every component with an object that maps the css selector with their hash equivalent, e.g. :

this is Button.module.css.{cjs,mjs}
```
'use client';
'use strict';

var classes = {"root":"m-77c9d27d","inner":"m-80f1301b","loader":"m-a25b86ee","label":"m-811560b9","section":"m-a74036a","group":"m-80d6d844"};

module.exports = classes;
//# sourceMappingURL=Button.module.css.cjs.map
```

Thanks !

**Comment by rtivital** [comment]
It is managed by rollup and postcss
